### PR TITLE
Slent fixm 244

### PR DESCRIPTION
**Pull request by slent-ctr**
In /docs/general-guidance/rules-for-absent-data.md:

I made minor edits to the text regarding the support for nillable elements in version 4.2.0, but did NOT update '4.2.0' to '4.3.0' since these changes were made in 4.2.0 and since then the support for nillable elements is more or less steady state. Perhaps this language should be removed?

---------------------------------------------------

IN /docs/how-to-create-application/initial-download-and-setup.md:

In step one of the instructions, I updated the link to point to the download page (https://fixm.aero/release.html?rel=Basic-Msg-1.1.0&ret=app), rather than directly to the .zip file.

### CODE REVIEW

**Review by cmcgowan-CTR — APPROVED** [review]

**Review by CPiroli-CTR — CHANGES_REQUESTED** [review]
There was talk about whether we wanted to leave the "How To" sections working with older versions of FIXM (since the version is irrelevant to the lessons being taught) or whether we wanted to update them to the latest version.  This change updates this "How To" to the latest version but doesn't change the instructions so that it matches how to build the schemas with the changes present in the latest version.  We should either revert this change (if we aren't going to update these sections) or fix it to work for 4.3.0 (which modified the schema build path).

**Review by CPiroli-CTR — COMMENTED** [review]
Not sure I oppose removing the pre-filled selections.  Just wanted to highlight the change to make sure we don't want these anymore.

**Review by CPiroli-CTR — COMMENTED** [review]
Same comment on "How To" sections.

**Review by CPiroli-CTR — COMMENTED** [review]
Link and test don't agree (text should be updated to 1.1.0) for FF-ICE Message change.

File: docs/home.md [hunk 1]
```diff
@@ -147,4 +147,4 @@ This sites aims to build a "community knowledge" about the implementation of FIX
 
 [OneSky Online]: https://ext.eurocontrol.int/
 [OneSky Teams]: https://ost.eurocontrol.int/Pages/default.aspx
-[SUBMIT CONTENT]: https://www.fixm.aero/content/contact.pl?category=Technical&version=Other&versionOther=FIXM%20User%20Manual&details=Describe%20proposed%20content%20here
+[SUBMIT CONTENT]: https://fixm.aero/contact.pl
```
Comment (CPiroli-CTR) [inline docs/home.md:150]: Are we sure we want to remove all the preset values?

**Pull request merged**